devel/ci/Dockerfile-pip

```diff
@@ -72,4 +70,7 @@ RUN rm -rf /bodhi/docs/_build
 RUN rm -rf /bodhi/docs/developer/docblocks
 RUN sed -i '/pyramid_debugtoolbar/d' setup.py
 RUN sed -i '/pyramid_debugtoolbar/d' devel/development.ini.example
+# The Makefile is set to use sphinx-build-3, but the pip installed sphinx is just called
+# sphinx-build.
+RUN sed -i 's/sphinx-build-3/sphinx-build/' docs/Makefile
```

Comment:
Couldn't we change this in the Makefile ? since the Makefile is in git.

Reply:
@cverna the problem is that most of the containers do need to call ```sphinx-build-3``` but this one ```pip``` container needs to call ```sphinx-build```. I decided to put ```sphinx-build-3``` in the Makefile since the Fedora 27, 28, 29, and Rawhide containers will all call it that, but that requires this exception in here in the pip container.

Reply:
ok that works for me :+1: